### DISCUSSION TITLE
Random Orientation Sampling

### DISCUSSION THREAD

**Discussion by darrencpagan** [neperfepx/neper#787] [Q&A] [answered]
It appears that when orientations are randomly sampled during the tesselation process, the same list of orientations is returned even if the tesselation seed is changed. Is there an option to resample a random orientation distribution so that when a new polycrystal is made, the grain morphology and the orientation list are both varied, not just the grain morphology? @mkasemer

**Comment by rquey** [comment, marked as answer]
The orientations should change with `-id`. A quick test on `neper - T -n 10 -id 1` vs `neper -T -n 10 -id 2` shows that the orientations differ.

Do you have an example where it does not work?

**Comment by darrencpagan** [comment]
I think this was an error are my part. Thanks for the response.
